modules/core/src/main/scala/com.snowplowanalytics.snowplow.snowflake/processing/Processing.scala

```diff
@@ -285,13 +302,23 @@ object Processing {
     batch: BatchAfterTransform
   ): F[BatchAfterTransform] =
     withWriteAttempt(env, batch) { notWritten =>
+      val setLatency =
+        // best case branch 2 - all remaining events left from attempt 1 are inserted
```

Comment:
Is there only one retry attempt? What happens when we have partial failure on the second attempt?

Reply:
partial failure is undefined for now, according to spec

Reply:
Oh yes - I forgot that but I did read it in the spec. Thanks